src/assets/css/infobox.css

```diff
@@ -0,0 +1,7 @@
+.infobox a {
+    color: #248bd7;
+}
+
+.infobox a:visited {
+    color: #248bd7;
+}
```

Comment:
I just realized that the other text in the infobox is using #585858, can you add that style as well?